lib/app/layouts/settings/pages/theming/avatar/avatar_crop.dart

```diff
@@ -23,7 +24,9 @@ class _AvatarCropState extends OptimizedState<AvatarCrop> {
   Uint8List? _imageData;
   bool _isLoading = true;
 
-  void onCropped(Uint8List croppedData) async {
+  void onCropped(Uint8List sizedData) async {
+    var image = img.copyResize(img.decodeImage(sizedData)!, width: 570, height: 570);
+    var croppedData = img.encodePng(image);
```

Comment:
imessage requires specific avatar formats

Reply:
Thia avatar is not sent to the server for anything but this change doesnt really matter that much I guess. Are you somehow using this avatar in rustpush?

Reply:
![image](https://github.com/BlueBubblesApp/bluebubbles-app/assets/50850142/09e55dc8-6030-4b45-a514-a130e515957b)